python/src/opendp/typing.py

```diff
@@ -368,6 +368,11 @@ def parse_or_infer(
         raise UnknownTypeException("either type_name or public_example must be passed")  # pragma: no cover
 
     def substitute(self: Union["RuntimeType", str], **kwargs):
+        '''
+        TODO
```

Comment:
@Shoeboxam , could you suggest a docstring here?

Reply:
Substitutes any generic type parameters according to the passed keyword arguments.